packages/core/src/transforms/resize.ts

```diff
@@ -43,8 +45,15 @@ export const resize: TransformFactory<ResizeOptions> = (config) => {
   const width = parseInt(config.width || config.w || '')
   const height = parseInt(config.height || config.h || '')
   const aspect = parseAspect(config.aspect || config.ar || '')
+  const withoutEnlargement = config.withoutEnlargement === '' || config.withoutEnlargement === 'true'
+  const withoutReduction = config.withoutReduction === '' || config.withoutReduction === 'true'
 
-  if (!width && !height && !aspect) return
+  if (
+    (!width && !height && !aspect) ||
+    (config.withoutEnlargement && !withoutEnlargement) ||
+    (config.withoutReduction && !withoutReduction)
```

Comment:
I think you can omit the `config.withoutEnlargement` and `config.withoutReduction` here since you already took them into account above right?